### PR TITLE
Updated API to 0.57.3

### PR DESCRIPTION
**Pull request by Skorfius**
Pgoapi update

### CODE REVIEW

File: pogom/search.py [hunk 1]
```diff
@@ -353,7 +353,7 @@ def search_overseer_thread(args, new_location_queue, pause_bit, heartb,
     account_queue = Queue()
     threadStatus = {}
     key_scheduler = None
-    api_version = '0.57.2'
+    api_version = '0.57.3'
```
Comment (SkOODaT) [inline pogom/search.py]: pretty sure you only change this when they force the new version.... doing this now would cause scanners to not work

Comment (BrownSlaughter) [inline pogom/search.py]: It is changed to what ever version we have up to date, this is to stop it using one we do not have yet.

Comment (myntath) [inline pogom/search.py]: I can confirm this currently stop scanners working. I tried these changes now and it paused because the api version didn't match.

Comment (BrownSlaughter) [inline pogom/search.py]: ah yes you are correct because of the check it does online against forced, this probably wont get merged until forced anyway

Comment (Obihoernchen) [inline pogom/search.py]: What's about changing the watchdog to check whether api_version is >= forced api?

**Comment by FrostTheFox** [comment]
a167384815c1c9a03068ce74ba3396979e1fb166

File: pogom/search.py (hunk 1 above)
Comment (Skorfius) [inline pogom/search.py]: ^that would be nice

**Pull request closed without merging**